### ISSUE TITLE
File annotations button flickering

### ISSUE THREAD

**Issue by ATrogolo** [microsoft/vscode#118413]
Issue Type: <b>Bug</b>

Hi,
I'm experimenting a bug in the last month with the button "File annotations" in the upper [toolbar](https://postimg.cc/CRtj4tH0)
When I click the button it will open the menu, and, after a couple of milliseconds, it closes itself. 
[Here a video sample](https://www.loom.com/share/90134de0bbb44dc79925b7aa8c438594)

Could you check this and help me to figure it out?

Thanks,
Andrea

VS Code version: Code 1.54.1 (f30a9b73e8ffc278e71575118b6bf568f04587c8, 2021-03-04T22:38:31.419Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.90GB (7.98GB free)|
|Process Argv|--crash-reporter-id ea1d18af-7e7a-48b2-b8ea-241bce2ef94b|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|12.0.1
npm-intellisense|chr|1.3.1
bracket-pair-colorizer|Coe|1.0.61
vscode-eslint|dba|2.1.14
githistory|don|0.6.14
gitlens|eam|11.3.0
vsc-material-theme|Equ|33.1.2
vsc-material-theme-icons|equ|1.2.2
prettier-vscode|esb|5.9.2
vs-code-xml-format|fab|0.1.5
githd|hui|2.2.4
vscode-styled-components|jpo|1.5.0
theme-monokai-pro-vscode|mon|1.1.18
vsliveshare|ms-|1.0.3912
vetur|oct|0.33.1
emoji|Per|1.0.1
material-icon-theme|PKi|4.5.0
markdown-preview-enhanced|shd|0.5.16
vscode-spotify|shy|3.2.1
language-stylus|sys|1.11.0
shellcheck|tim|0.13.2
json2yaml|tux|0.2.0
quokka-vscode|Wal|1.0.345
save-and-run|wk-|0.0.22

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30256636
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
dockersubset:30265998
pythontb:30265425
vspre833:30267464

```

</details>

**Comment by sbatten** [comment]
@ATrogolo I cannot reproduce this myself, but I'm happy to work with you to figure out the repro. As a first step, can you try disabling all of you extensions except for gitlens to see if this reproduces?

**Comment by ATrogolo** [comment]
Hi @sbatten , 
I tried to identify the cause of the bug using the VSCode feature `Extension bisect`.
It reports the extension causing the problem is [Quokka.js](https://marketplace.visualstudio.com/items?itemName=WallabyJs.quokka-vscode) and I opened them an issue: https://github.com/wallabyjs/quokka/issues/589

Andrea

**Comment by sbatten** [comment]
@ATrogolo thanks for digging on this. I installed that extension and can reproduce the issue to do more debugging myself, thanks again

**Comment by smcenlly** [comment]
I'm from the team that create's the Quokka extension. I don't think that this is a bug in Quokka, rather it's something in VS Code. @sbatten - please let us know if you find otherwise.

**Comment by smcenlly** [comment]
@sbatten do you have an update on this? We have had another one of our users report the problem.

**Comment by ATrogolo** [comment]
Hi,
I've checked the issue with the last version of VsCode (and Quokka.js extension) but it still doesn't work.
Any news about a fix?

**Comment by sbatten** [comment]
We made a change this milestone around how these submenus are rendered, I can no longer repro and thus believe this to be fixed. @smcenlly can you verify

**Comment by ATrogolo** [comment]
Ok, thanks
So it will be shipped with the next release (> 1.59.1)?

**Comment by sbatten** [comment]
@ATrogolo or in the latest insiders
